### ISSUE TITLE
Error in exit name

### ISSUE THREAD

**Issue by bortzmeyer** [ethereum/guide#1]
The guide for the interactive console is great but it seems that the name of the exit function is web3.admin.eth.exit not web3.admin.exit.

```
> web3.eth.exit()
(shell):1: TypeError: Object #<Object> has no method 'exit'
web3.eth.exit()
         ^
TypeError: Object #<Object> has no method 'exit'
    at (shell):1:10
Error: Uncaught TypeError: Object #<Object> has no method 'exit'

> web3.admin.eth.exit()
true
 ⚡   22:51:18|eth  Stop worker 492 ms
  ℹ  22:51:18|eth  Closing blockchain DB
 ⚡   22:51:18|eth  Terminate worker 307 ms
 ⚡   22:51:18|eth  Terminate worker 328 ms
  ℹ  22:51:18|eth  Closing state DB
 ⚡   22:51:19|eth  Worker stopping 925 ms
 ⚡   22:51:19|ethsync  Worker stopping 1485 ms
 ⚡   22:51:19|p2p  Worker stopping 1505 ms
> web3.admin.
`̀``


```

**Comment by LefterisJP** [comment]
The guide has been updated to reflect the upcoming 1.0 release which will contain the new and updated name. You can check it out yourself too by building by source from the develop branch.

One other such change is that `web3.admin.web3.setVerbosity()` becomes `web3.admin.setVerbosity()`

**Comment by bortzmeyer** [comment]
So, the guide documents the future, post-Frontier?

**Comment by LefterisJP** [comment]
The guide documents 1.0. We have not yet released 1.0, only Go did. Our 1.0 release will be very shortly. I apologize for any confusion.
